optaplanner-core/src/test/java/org/optaplanner/core/impl/score/director/drools/testgen/fact/TestGenValueFactTest.java

```diff
@@ -40,7 +38,7 @@ public void initialization() {
 
         StringBuilder sb = new StringBuilder(100);
         fact.printInitialization(sb);
-        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();\n", sb.toString());
+        assertEquals("    private final TestdataValue testdataValue_321 = new TestdataValue();" + System.lineSeparator(), sb.toString());
```

Comment:
It's because TestGen classes actually use `System.lineSeparator()` when printing code. My intention was to produce output in a format which is native to the runtime environment.

The restriction you are mentioning is there to avoid mess in the files that are version controlled. TestGen output (the reproducing test) is not meant to be version controlled so it doesn't matter much what line endings are used there.

So we just need to make a trade off between two extreme situations:
1. A Windows user opens the generated test in some ancient editor that doesn't support linux endings. The user is not happy. This won't happen with the current solution.
2. The Windows user is happy and attaches the file to a JIRA, a Drool developer copy-pastes part of that file and IF he uses an IDE that doesn't convert line endings, he may commit a test with Windows line endings. This can happen with the current solution.

I can hardly imagine a Windows user/developer using a text editor/IDE that doesn't support `\n`s nowadays so I think it might actually be more consistent to use `\n`s under all circumstances when printing the generated reproducer. What do you think?